lib/travis/build/script/node_js.rb

```diff
@@ -222,7 +224,16 @@ def iojs_3_plus?
 
           def npm_install(args)
             sh.fold "install.npm" do
-              sh.cmd "npm install #{args}", retry: true
+              sh.if "$(vers2int `npm -v`) -ge $(vers2int #{NPM_CI_CMD_VERSION})" do
+                sh.if "-f npm-shrinkwrap.json -o -f package-lock.json" do
+                  sh.cmd "npm ci #{args}", retry: true
+                else
```

Comment:
This should have been `end <newline> sh.else do`. My bad 😬

Reply:
That's my bad. Ruby isn't my strong suit.